lib/BackgroundJob/EmailNotification.php

```diff
@@ -61,10 +61,10 @@ class EmailNotification extends TimedJob {
 	 * @param bool|null $isCLI
 	 */
 	public function __construct(
-		MailQueueHandler $mailQueueHandler = null,
+		?MailQueueHandler $mailQueueHandler,
```

Comment:
```suggestion
		?MailQueueHandler $mailQueueHandler = null,
```

We should be able to leave `= null` as the default value. Then we do not have to change code that calls the constructor.

(same for other lines below)

Reply:
wow